src/Uno.UI.Tests/Windows_UI_Xaml/Given_ThemeResource.cs

```diff
@@ -14,14 +14,27 @@ namespace Uno.UI.Tests.Windows_UI_Xaml
 	public class Given_ThemeResource
 	{
 		[TestMethod]
-		public void When_System_ThemeResource()
+		public void When_System_ThemeResource_Light()
 		{
 			var app = UnitTestsApp.App.EnsureApplication();
+			app.ForceSetRequestedTheme(Windows.UI.Xaml.ApplicationTheme.Light);
 
 			var control = new Test_Control();
 			app.HostView.Children.Add(control);
 
 			Assert.AreEqual(Color.FromArgb(0xDE, 0x00, 0x00, 0x00), (control.TestTextBlock.Foreground as SolidColorBrush).Color);
 		}
+
+		[TestMethod]
+		public void When_System_ThemeResource_Dark()
+		{
+			var app = UnitTestsApp.App.EnsureApplication();
+			app.ForceSetRequestedTheme(Windows.UI.Xaml.ApplicationTheme.Dark);
+
+			var control = new Test_Control();
+			app.HostView.Children.Add(control);
+
+			Assert.AreEqual(Color.FromArgb(0xDE, 0xFF, 0xFF, 0xFF), (control.TestTextBlock.Foreground as SolidColorBrush).Color);
+		}
```

Comment:
Set the theme back to light at the end of this test, in case other tests are assuming that

Reply:
Oh right! Similar issue happened in other test, I could not wrap my head around why it sometimes randomly crashes until I realized this :-D .

Reply:
Done 👌